app/src/modelProviders/fine-tuned/getFireworksCompletion.ts

```diff
@@ -36,6 +36,12 @@ export async function getFireworksCompletion(
     throw new Error("FIREWORKS_API_KEY is required for Fireworks completions");
   }
 
+  if ((input.functions || input.tools) && input.stream) {
+    throw new Error(
+      `Error: Streaming completions are not supported for function calls. Model: ${fineTune.slug}`,
```

Comment:
"We don't currently support streaming for function calls. Please open an issue if you need this functionality! https://github.com/openpipe/openpipe"